exopy_qm/instruments/drivers/QuantumMachine.py

```diff
@@ -1,8 +1,17 @@
-import importlib
 import sys
+import logging
 from qm.QuantumMachinesManager import QuantumMachinesManager
 
-from exopy_qm.utils.dynamic_importer import *
+
+def requires_config(func):
+    def wrapper(self, *args, **kwargs):
+        if self.qmObj:
+            func(self, *args, **kwargs)
+        else:
+            log = logging.getLogger()
```

Comment:
I would suggest creating the logger at module level using `logger = logging.getLogger(__name__)`. This will avoid using an unnamed logger.